lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -22,6 +24,9 @@ def detach_volume(volume_id, options = {})
       sync_or_async_ems_operation(options[:sync], "detach_volume", [volume_id])
     end
 
+    # this implents the AvailabilityMixin interface
+    # for backwards compatibility in customer scripts
+    # prefer using supports_resize? method instead
```

Comment:
Should this also `Vmdb::Deprecation.deprecate_methods(self, :validate_resize => :supports_resize?)` or something similar?